src/Umbraco.Cms.Api.Management/Controllers/Media/Collection/MediaCollectionControllerBase.cs

```diff
@@ -15,10 +19,20 @@ namespace Umbraco.Cms.Api.Management.Controllers.Media.Collection;
 [VersionedApiBackOfficeRoute($"{Constants.Web.RoutePath.Collection}/{Constants.UdiEntityType.Media}")]
 [ApiExplorerSettings(GroupName = nameof(Constants.UdiEntityType.Media))]
 [Authorize(Policy = AuthorizationPolicies.SectionAccessMedia)]
-public abstract class MediaCollectionControllerBase : ContentCollectionControllerBase<IMedia, MediaCollectionResponseModel, MediaValueResponseModel, MediaVariantResponseModel>
+public abstract class MediaCollectionControllerBase : ContentCollectionControllerBase<IMedia, MediaCollectionResponseModel, MediaValueResponseModel, MediaVariantResponseModel, DocumentCollectionResponseModel>
 {
+    private readonly SignProviderCollection _signProviders;
+
+    [ActivatorUtilitiesConstructor]
```

Comment:
This isn't needed here - only required on public constructors for controllers as it's used by .NET in mapping requests to controller action methods.